text2sql-frontend/src/router.tsx

```diff
@@ -52,7 +52,7 @@ let private_routes: RouteObject[] = [
   },
 ];
 
-if (process.env.NODE_ENV === "local") {
+if (process.env.NODE_ENV === "development") {
```

Comment:
the reason this was local is to distinguish between the dataline website (landing page) and the actual product, regardless of the dev/production state. @RamiAwar can you confirm this?